docs/user-guide/ze-working-with-uss-files.md

```diff
@@ -16,6 +16,21 @@
 
    ![Enter Path](../images/ze/ZE-path.gif)
 
+## Dragging and dropping USS files
+
+To select one or more files or folders in the tree and drag them to a folder or session to relocate them:
+
+1. Expand **UNIX SYSTEM SERVICES (USS)** in the **Side Bar**.
+2. Select the files or folders you want to move.
+   - Select multiple items by pressing the `Ctrl` key as you select each item.
+   - Select a continuous range of items by selecting the first item in the range. Then hold down the `Shift` key until selecting the last item in the range.
+3. Click and hold the region containing the selected items to start the drag process.
+4. As the items are selected, move your cursor to the desired folder or session. Release the mouse button to start the move operation.
+   - If the move operation might overwrite one or more files, an “overwrite confirmation" message displays. Select **Confirm** to relocate these items. Select **WHAT** to stop the move operation.
```

Comment:
@traeok Does the UI say **Confirm** or **Continue**? And what option does the UI provide for the user to cancel the move operation?

Reply:
The options for the UI are "Confirm" and "Cancel". The message says: "X is opened and has pending changes in the editor. By selecting 'Confirm', any unsaved changes will be lost." 

(X being the name of the file)

Reply:
@traeok -- updated the doc to reflect this -- thank you!